### DISCUSSION TITLE
Cannot download security onion source code

### DISCUSSION THREAD

**Discussion by mmorales0113** [Security-Onion-Solutions/securityonion#4009] [Unsupported Versions] [answered]
I'm having trouble downloading the security onion source code on Ubuntu. I get:

$ sudo apt-get source securityonion-all  
E: You put some 'source' URIs in your source.list

**Comment by TOoSmOotH** [comment, marked as answer]
Our source is here in github. We do not create any packages for Ubuntu in SO 2.

**Reply by mmorales0113** [reply]
Can you point me to the repo. I can't find it. Thank  you!!

**Reply by TOoSmOotH** [reply]
This is the repo
